core/types.go

```diff
@@ -27,6 +32,7 @@ type Authorization struct {
 	ID          string
 	URL         string
 	ExpiresDate time.Time
+	Order       *Order
```

Comment:
I was confused by some diff collapsing and thought this was a field on Registration. This is fine for now. Thanks!